app/components/ui/files/publish-modal/template.hbs

```diff
@@ -97,28 +99,37 @@
 </div>
 
 <div class="content">
-    <div class="ui form"></div>
+    <div class="ui form">
         <div class="grouped fields">
             <div class="field">
-                <div class="ui radio checkbox">
-                    <input type="radio" name="CC0" checked="checked">
-                    <label>Creative Commons Public Domain CCO <i class="info circle blue icon" ></i> </label> 
+                <div class="ui radio checkbox cc0" id='0'>
+                    <input type="radio" name="license-radio" checked="checked">
+                    <label><img class="ui small image left floated" src="https://d15omoko64skxi.cloudfront.net/wp-content/uploads/2014/01/cc-zero.png">
```

Comment:
I don't know how others feel, but maybe referencing images on external servers is likely to break at some point or annoy the respective image hosts. What do you think about including these images as formal assets in this repo?